manager/ui/war/plugins/api-manager/html/policyForms/ignored-resources.include

```diff
@@ -37,8 +37,8 @@
   <table class="table table-striped table-bordered">
     <thead>
       <tr>
-        <th width="30%" apiman-i18n-key="verb">Method</th>
-        <th width="70%" apiman-i18n-key="path">Path</th>
+        <th width="30%" apiman-i18n-key="path">Path</th>
+        <th width="70%" apiman-i18n-key="verb">Method</th>
```

Comment:
Done! Also forgot about the favicon change for the JSON schema viewer last week, so added that in here as well.